src/router/views/download.vue

```diff
@@ -0,0 +1,313 @@
+                                                    @click="setPreviewAction()">Show Preview
+                                            </button>
+                                            <sta-data-exporter
+                                                    :id="[ isLoading ? 'disabled' : 'enabled' ]"
+                                                    style="margin-bottom: 4px; margin-left: 4px"
+                                                    class="btn btn-primary"
+                                                    :json-fields="export_fields"
+                                                    :cb-function="setDownloadAction"
+                                                    :name="exportFileName"
+                                                    :before-loading="startProgress"
+                                                    :after-loading="endProgress"
+                                            />
+                                            <div v-show="isLoading"
+                                                 class="spinner-border spinner-border-sm text-primary"
+                                                 style="display:inline-block; margin-left: 5px; " role="status"
+                                                 aria-hidden="true"></div>
+                                            <sweet-modal ref="Unsuccessful" icon="error" title="Error happened...">
+                                                {{statusResult}}
+                                            </sweet-modal>
+                                            <div v-show="previewTableSettings.data.length" class="alert alert-primary"
+                                                 role="alert"
+                                                 style="margin-top: 4px; margin-bottom: 4px">
+                                                This preview table shows only a subset of the data
+                                            </div>
+                                            <div class="mt-2 mb-4"
+                                                 style="width: 100%; height: 600px; overflow-x: hidden">
+                                                <hot-table :settings="previewTableSettings"></hot-table>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </main-content>
+    </layout>
+</template>
+
+<script>
+    import {SweetModal} from 'sweet-modal-vue'
+    import Layout from '@layouts/main';
+    import MainContent from '@layouts/main-content';
+    import staRequests from './sta-request';
+    import HotTable from '@handsontable/vue';
+    import staHeaders from '@components/data-exporter/sta-headers.js';
```

Comment:
Might change to import {table} from '@components/data-exporter/sta-headers.js'; and replace `staHeaders.table` to `table` directly